arroyo-compiler-service/src/main.rs

```diff
@@ -74,9 +78,54 @@ pub async fn main() {
         base_path,
     };
 
-    let addr = "0.0.0.0:9000".parse().unwrap();
+    let args = std::env::args().collect::<Vec<_>>();
+    match args.get(1) {
+        Some(arg) if arg == "start" => {
+            start_service(service).await;
+        }
+        Some(arg) if arg == "compile" => {
+            let path: object_store::path::Path = args
+                .get(2)
+                .expect("Usage: ./compiler_service compile <query-req-path>")
+                .to_string()
+                .try_into()
+                .unwrap();
+
+            let query = service
+                .object_store
+                .get(&path)
+                .await
+                .expect("Failed to read file from object store")
+                .bytes()
+                .await
+                .expect("Failed to read file from object store");
+
+            let query = CompileQueryReq::decode(&*query).expect("Failed to decode query request");
+
+            let resp = service.compile(query).await.unwrap();
+            println!(
```

Comment:
should this be an info!()?

Reply:
It's writing json to std out so that it can be consumed by whatever is calling it. If it were an info!, that would send it to the logging system instead.